components/WorkDatesDisplay/WorkDatesDisplay.js

```diff
@@ -28,6 +28,29 @@ export default function WorkDatesDisplay({ workDates, onDateRemove }) {
       ) : (
         <p>No Vacations yet!</p>
       )}
+      <h4>Your Days-Off:</h4>
+      {workDates && workDates.length > 0 ? (
+        <ul>
+          {workDates.map((workDate) => (
+            <li key={workDate._id}>
+              {workDate.daysOff.map((dayOff, index) => (
+                <Fragment key={index}>
+                  <Dates>
+                    Day Off: {new Date(dayOff).toLocaleDateString()}
+                  </Dates>
+                  <DeleteButton
+                    onClick={() => onDateRemove(index, workDate._id, dayOff)}
+                  >
```

Comment:
The deleted tag will be removed from the list only after you reload the page, please add a logic here.

Reply:
Yes, I really have this problem despite trying to use logic that could solve it. I will try to do better.
However, this capability is not described in the user story :D

Reply:
Cool